async-request-reply/src/dx-excshell-1/actions/polling/index.js

```diff
@@ -0,0 +1,48 @@
+
+/**
+ * This action is used to poll the status of a long running action
+ * It is invoked with an activationId as a parameter to poll
+ * or it will invoke the long-running action if no activationId is provided
+ * it will return a 503 Service Unavailable if the long running action is not complete
+ * It will return the result of the long running action when it is complete 200 OK
+ */
+
+let openwhisk = require("openwhisk");
```

Comment:
add a comment why this is not in package.json, since it is already default included in runtime